frontend/web/components/Dateline.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from 'react';
+import { getCookie } from '../lib/cookie';
+import dateformat from 'dateformat';
+
+const dtFormatGU = (date: Date, edition: string): string => {
+    let datetimeFormatted: string = dateformat(date, 'ddd d mmm yyyy HH:MM Z');
+    if (edition === 'UK') {
+        datetimeFormatted = datetimeFormatted.replace('GMT+0100', 'BST');
+    }
+    return datetimeFormatted;
+};
+
+const defaultEdition = 'UK';
+
+class Dateline extends Component<
+    { capiDate: Date },
+    { capiDate: Date; edition: string }
+> {
+    constructor(props: any) {
```

Comment:
Can the props take the same type as the `Component`?
```
{ capiDate: Date; edition: string }
```

Reply:
Certainly. Thanks for spotting this!